norfair/drawing.py

```diff
@@ -375,6 +375,53 @@ def draw_tracked_boxes(
     return frame
 
 
+class Paths:
+    def __init__(self, points_of_interest=None, thickness=None, color=None, radius=None, attenuation=0.01):
+        if points_of_interest is None:
```

Comment:
Arguments which are functions tend to be called something like `<name>_fn` so its clear that they're functions, or in case they aren't named like this they have a very obvious function name, usually including a verb like `get_<something>` that makes it clear they're functions. I would change this to something like `get_points_to_draw`, as there is no need to make it generic by saying `of_interest` instead of `to_draw`.